official-antistasi-community/A3-Antistasi
# [Change] ] Vanilla Templates, added additional DLC conditionals

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Utilizing enabledDLC switches for DLC vehicles and weapons
Tweaking inconsistencies between templates


### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

Loaded each template, checked RPT for non-obvious errors.

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Check DLC boxes, Load template, observe changes.
Provide opinion on changes to the vehicle lineups.

********************************************************
Notes:
Changes:
1. [x] DLC conditionals for vehicles
2. [x] Matched vehicle templates (ie Nato tropical only used HMG mrap while others used both)
3. [x] Moved DLC vehicles who had possible non-dlc standins to relevant DLC toggle*
4. [x] CSAT DLC weaponry.
5. [x] NATO DLC weaponry.
6. [x] AAF DLC weaponry.
7. [x] LDF DLC weaponry.

*Please review and give feedback on this point in particular.
